modules/reference/pages/glossary.adoc

```diff
@@ -20,11 +34,21 @@ An internal TigerGraph characteristic of a TigerGraph cluster that determines ho
 
 A characteristic of the cluster design that will determine the number of copies of data that will be stored in the cluster.
```

Comment:
This definition is wrong. The operative definition: When a user creates a new solution, one of the first steps is the following:
```
Select an Instance Type
Choose a virtual machine based on your data size and compute performance needs.
```
An instance is one virtual machine or virtual server.  See the definition of **cluster**: `A group of servers connected together`. This could say `a group of instances connected together.`